.github/workflows/ci-dotnet.yml

```diff
@@ -0,0 +1,21 @@
+name: Kavita - CSharp
```

Comment:
Why do we need this when we are already building in the main workflow? This would increase our build to 3 times in a single PR, which is the opposite of the ask in the issue (builds are too slow)

Reply:
The idea was to split out different things into different jobs for faster feedback.  Something like this would be "faster" while there would be a main build once things are merged that would do a docker build and other stuff if needed.

Maybe I misread the requirement for the issue but I thought this was what was being asked for with the usage of certain github actions